Kickstarter-iOS/Views/Controllers/DashboardViewController.swift

```diff
@@ -114,6 +120,12 @@ internal final class DashboardViewController: UITableViewController {
         element?.updateData(data)
     }
 
+    self.viewModel.outputs.goToMessages
+      .observeForControllerAction()
+      .observeValues { [weak self] project in
+        self?.goToMessages(project: project)
+      }
```

Comment:
Ah, just indentation doesn't match the others. Can cut and paste to let Xcode reindent :)